qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/jdt/internal/QuteJavaConstants.java

```diff
@@ -39,8 +39,14 @@ public class QuteJavaConstants {
 
 	public static final String TEMPLATE_EXTENSION_ANNOTATION = "io.quarkus.qute.TemplateExtension";
 
+	public static final String TEMPLATE_DATA_ANNOTATION = "io.quarkus.qute.TemplateData";

```

Comment:
Please remove this constants which is already declared (see on the bottom of the file afyter // @TemplateData comment